nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -437,6 +474,11 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Gas|OCGT:
                 for a typical Open Cycle Gas Turbine power plant
    unit: "%"
 
+Rate Automatic Frequency Restoration Reserve|Electricity|Gas|CCGT:
+   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
+                for a typical Open Cycle Gas Turbine power plant
```

Comment:
...for a typical Combined Cycle Gas Turbine power plant.

Reply:
:-)